packages/elements/src/components/ui/action-menu/ActionMenuItem.tsx

```diff
@@ -49,9 +50,13 @@ export const ActionMenuItem = forwardRef<
   return (
     <button
       {...props}
-      className={cx(styles.actionMenuItem, styles[variant], className)}
+      className={cx(
+        styles.actionMenuItem,
+        styles[disabled ? "disabled" : variant],
+        className
+      )}
       onKeyDown={onKeyDown}
-      onClick={props.disabled ? undefined : onClickHandler}
+      onClick={disabled ? undefined : onClickHandler}
```

Comment:
This causes the underlying button to not get the disabled prop, keep using `props.disabled` :)

Reply:
I was wondering why it was so explicit, fixed.